apps/quarkus-json/src/main/java/org/acme/Record.java

```diff
@@ -0,0 +1,26 @@
+package org.acme;
+
+import java.util.List;
+import java.util.Map;
+
+public class Record {
+    public String s0;
+    public String s1;
+    public String s2;
+    public String s3;
+    public String s4;
+    public String s5;
+    public String s6;
+    public String s7;
+    public String s8;
+    public String s9;
+    public int i0;
+    public int i1;
+    public int i2;
+    public int i3;
+    public int i4;
+    public int i5;
+    public int i6;
```

Comment:
Would it make sense to include some more types to increase coverage?

Reply:
Not sure, I will take a look what the serializer/deserializer does with e.g. BigInt etc. Perhaps calendar dates...some more complex types. The core idea is string processing though. Maybe we can have more small apps?